pylib/mozhginfo/mozhginfo/pushlog_client.py

```diff
@@ -0,0 +1,187 @@
+#! /usr/bin/env python
```

Comment:
This file should have the MPL 2 boilerplate header. 

# This Source Code Form is subject to the terms of the Mozilla Public
# License, v. 2.0. If a copy of the MPL was not distributed with this
# file, You can obtain one at http://mozilla.org/MPL/2.0/.